mocks/auth-server.js

```diff
@@ -3,76 +3,22 @@
 const compression = require('compression')
 const express = require('express')
 const jwt = require('jsonwebtoken')
+const { find } = require('lodash')
 
 const { decodeAccessToken } = require('../common/lib/access-token')
 const {
-  E2E: { ROLES },
+  E2E: { USERS },
 } = require('../config')
 
-const roleConfig = {
-  COURT: {
-    locations: ['ABDRCT', 'BDFRCT'],
```

Comment:
We're already pulling in config from the main app (see line `9`) so there's not much difference in where this config lives.

I'm not trilled about having test config in the main app config, but I'm not seeing a cleaner way of dealing with this at the moment.

Reply:
Yeah, good point. I did think of moving this just to the auth server but some of the config is pulled in in various places in the e2e tests. So config still seemed the most logic place...